src/test/test-execute.c

```diff
@@ -218,6 +219,41 @@ static void start_parent_slices(Unit *unit) {
         }
 }
 
+static bool can_use_unprivileged_userns(void) {
+        pid_t pid;
+        int r;
+
+        r = safe_fork("(sd-test-check-unpriv-userns)",
+                      FORK_RESET_SIGNALS |
+                      FORK_CLOSE_ALL_FDS |
+                      FORK_DEATHSIG_SIGTERM,
```

Comment:
i'd probably switch to SIGKILL here. we don't need to be friendly here, the child does not leave anything around that might need cleanup if we just do SIGKILL.